parsers/uniprot/uniprot_test.go

```diff
@@ -0,0 +1,13 @@
+package uniprot
+
+import (
+	"fmt"
+)
+
+func ExampleReadUniprot() {
+	entries, _, _ := ReadUniprot("data/uniprot_sprot_mini.xml.gz")
+
+	entry := <-entries
+	fmt.Println(entry.Accession[0])
+	// Output: P0C9F0
+}
```

Comment:
Can you write a test for `xml.go`'s Marshall functionality? I realized it'd be faster if you did this @Koeng101.

Reply:
No. Just removed em